modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/ParameterDescriptor.java

```diff
@@ -65,16 +64,6 @@ void setDefaultValue(Object defaultValue)
         this.defaultValue = defaultValue;
     }
 
-    boolean isHidden()
```

Comment:
not used anymore. Haven't for a while but only now I noticed. I didn't see the point at removing it in a separate PR, create an issue, etc